test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -266,7 +266,7 @@ private static String convertPersonToAddCommandString(ReadOnlyPerson person) {
                 + person.getName().fullName
                 + (person.getPhone().isPrivate() ? " pp/" : " p/") + person.getPhone().value
                 + (person.getEmail().isPrivate() ? " pe/" : " e/") + person.getEmail().value
-                + (person.getAddress().isPrivate() ? " pa/" : " a/") + person.getAddress().value;
+                + (person.getAddress().isPrivate() ? " pa/" : " a/") + person.getAddress().toString();
```

Comment:
how do you generate the string representation? now that i have suggested to remove the `value` variable, you need a way to generate the original address from the components.